snapshot-labs/snapshot
# fix: shorten long vote balance

### Issues
Shorten long(> 10 length) vote balance, show full in tooltip.

Fixes https://github.com/snapshot-labs/snapshot/issues/3744

### Changes 
1. Added balanceFormatted computed to SpaceProposalVotesListItem.vue

### How to test
1. Go to this proposal https://snapshot.org/#/arbitrumfoundation.eth/proposal/0x373dfa89fc9c5ccba8ed83fa3fa4f233edd270075b5f8f4f3902b408318d9d17 and sort asc by vp to see.

### Self-review checklist
